cmd/minikube/cmd/start.go

```diff
@@ -1314,6 +1318,25 @@ func validateListenAddress(listenAddr string) {
 	}
 }
 
+// This function validates that the --ports are not below 1024 for the host and not outside range
+func validatePorts(ports []string) {
+	for _, portDuplet := range ports {
+		for i, port := range strings.Split(portDuplet, ":") {
+			p, err := strconv.Atoi(port)
+			if err == nil {
```

Comment:
Small nit, but if you did `if err != nil` instead there would be less nesting and would look a bit cleaner.

```
if err != nil {
    exit.Message(...)
}
if p > 65535 { ... }
if p < 1024 { ... }
```

Reply:
And that's exactly what I should have done :)